docs/faq.rst

```diff
@@ -220,3 +220,27 @@ After getting the resources you'll need, you will just need to make sure your
 runtime environment is able to access the filesystem, at the location of your
 *TemplateFlow home* directory.
 If you are a Singularity user, please check out :ref:`singularity_tf`.
+
+How do I select only certain files as input to fMRIPrep?
+---------------------------------------------------------------------
+
+Using the `--bids-filter-file` flag, you can pass fMRIPrep a JSON file that
```

Comment:
The two hyphens are currently being merged into a dash
https://12540-53175327-gh.circle-artifacts.com/0/tmp/src/fmriprep/docs/_build/html/faq.html#how-do-i-select-only-certain-files-as-input-to-fmriprep

this should fix it
```suggestion
Using the ``--bids-filter-file`` flag, you can pass fMRIPrep a JSON file that
```

Reply:
Wow, I thought I was detailed-oriented, but did _not_ catch that--thank you!